docs/site/src/pages/reference/console/ui-overview.mdx

```diff
@@ -53,7 +53,7 @@ workspaces.
     </tr>
     <tr>
       <td>[Tasks](/reference/device-drivers)</td>
-      <td>Tasks are the current read and write tasks happening on devices connected to
+      <td>Tasks are the current read and write tasks running on devices connected to
```

Comment:
for all the descriptions in this table - do we need the "x are" at the front of each desciption?